packages/tpetra/core/src/Tpetra_Details_WrappedDualView.hpp

```diff
@@ -98,28 +98,55 @@ using enableIfConstData = std::enable_if_t<hasConstData<DualViewType>::value>;
 template <typename DualViewType>
 using enableIfNonConstData = std::enable_if_t<!hasConstData<DualViewType>::value>;
 
+/* sync_host functions */
+  
 template <typename DualViewType>
 enableIfNonConstData<DualViewType>
 sync_host(DualViewType dualView) {
   // This will sync, but only if needed
   dualView.sync_host();
 }
 
+template <typename DualViewType>
+enableIfNonConstData<DualViewType>
+sync_host(const typename DualViewType::t_host::execution_space& exec, DualViewType dualView) {
```

Comment:
```suggestion
sync_host(const typename DualViewType::t_host::execution_space& exec, DualViewType& dualView) {
```
(Not sure why you want to take by value in this case...)